hoot-core-test/src/test/cpp/hoot/core/io/ServiceHootApiDbWriterTest.cpp

```diff
@@ -64,46 +64,45 @@ class ServiceHootApiDbWriterTest : public CppUnit::TestFixture
 
 public:
 
-  static QString userEmail() { return "ServiceHootApiDbWriterTest@hoottestcpp.org"; }
+  QString userEmail() { return QString("%1.ServiceHootApiDbWriterTest@hoottestcpp.org").arg(testName); }
+  QString userName()  { return QString("%1.ServiceHootApiDbWriterTest").arg(testName); }
 
-  void compareRecords(QString sql, QString expected, QVariant v1 = QVariant())
```

Comment:
`compareRecords` moved to the bottom and `setUpTest` and `tearDown` move to the top.